src/tribler-core/tribler_core/tests/test_notifier.py

```diff
@@ -1,14 +1,38 @@
+# pylint: disable=redefined-outer-name
+import pytest
+
 from tribler_common.simpledefs import NTFY
 
 from tribler_core.notifier import Notifier
 
 
-def test_notifier():
-    notifier = Notifier()
+@pytest.fixture
+def notifier():
```

Comment:
```suggestion
@pytest.fixture(name="notifier")
def fixture_notifier():
```

Should avoid `# pylint: disable=redefined-outer-name`.

Reply:
@qstokkink 👍